static/app/components/autoComplete.tsx

```diff
@@ -311,7 +335,7 @@ class AutoComplete<T extends Item> extends React.Component<Props<T>, State<T>> {
 
     // when this component is in virtualized mode, only a subset of items will be passed
     // down, making the array length inaccurate. instead we manually pass the length as itemCount
-    const listSize = this.itemCount || this.items.size;
+    const listSize = this.itemCount ?? this.items.size;
```

Comment:
A bit more correct, we don't want `itemCount` 0 defaulting to `item.size` I believe